### PR TITLE
Update testsalesforce flow

### PR DESCRIPTION
**Pull request by yloson**
this pr is for changing current salesforce flow to work w/o global search + change to hour picking (instead of hard coded value to first available value)

### CODE REVIEW

File: src/test/java/base/Utils.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package base;
+
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class Utils {
+
```
Comment (pazaricha) [inline src/test/java/base/Utils.java]: minor: remove extra-line space

File: src/test/java/base/Utils.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    public String convertStreamToString(InputStream is) throws Exception {
```
Comment (pazaricha) [inline src/test/java/base/Utils.java]: i think `is` is a saved keyword? perhaps make the name more explicit: `input_stream`

File: src/test/java/base/Utils.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        BufferedReader reader = new BufferedReader(new InputStreamReader(is));
+        StringBuilder sb = new StringBuilder();
```
Comment (pazaricha) [inline src/test/java/base/Utils.java]: also here, I personally prefer verbosity for variable names: `string_builder`

File: src/test/java/tests/TestSalesfoce.java [hunk 4]
```diff
@@ -13,48 +14,48 @@
...
 
     @BeforeTest
     public void before() throws Exception {
         /* Generating new lead */
         SalesforceApi salesforceApi = new SalesforceApi();
         HttpResponse response = salesforceApi.generateLead();
+        InputStream instream = response.getEntity().getContent();
+        Utils utils = new Utils();
+        String jsonResponse = utils.convertStreamToString(instream);
+        lead_sfid = utils.extractResponseValueByKey(jsonResponse, "", leadSalesforceID);
```
Comment (pazaricha) [inline src/test/java/tests/TestSalesfoce.java]: I would change var name to `lead_sf_id`

File: src/test/java/base/Utils.java (hunk 1 above)
Comment (yloson) [reply src/test/java/base/Utils.java]: all classes are separate by line, i find it clearer to view

File: src/test/java/pageObjects/ManageTours.java [hunk 5]
```diff
@@ -32,14 +33,14 @@
     private By tourNameInGrid = By.cssSelector("div[data-field='Name'] a");
     private By tourInfoSection = By.cssSelector(".slds-hint-parent");
 
-    public void bookTour(String numFullTimeEmp, String building, String tourHour) {
+    public void bookTour(String numFullTimeEmp, String building) {
```
Comment (okeren-ww) [inline src/test/java/pageObjects/ManageTours.java]: what is the `emp` part of `numFullTimeEmp`?

I would rename the variable to be more descriptive

Comment (yloson) [reply src/test/java/pageObjects/ManageTours.java]: emp stands for employees - i changed it

**Pull request merged**